content/source/docs/cloud/workspaces/managing-variables.html.md

```diff
@@ -57,6 +66,15 @@ To delete a variable:
 1. Click the ellipses next to the variable you want to delete and select **Delete**.
 1. Click **Yes, delete variable** to confirm your action.
 
+## Loading Variables from Files
+
+You can set variable values by providing any number of [files ending in `.auto.tfvars`](/docs/language/values/variables.html#variable-files) to workspaces that use Terraform 0.10.0 or later. When you trigger a run, Terraform automatically loads and uses the variables defined in these files. You can only do this with files ending in `auto.tfvars`; Terraform Cloud does not recognize other types of `.tfvars` files. If any variable from the workspace has the same key as a variable in the file, the workspace variable overwrites variable from the file.
```

Comment:
What do you think of this? I want to help users understand how this is different than run-specific variables above. Do you think this is clear?  It's all so nuanced!

```suggestion
You can set [Terraform variable](/docs/cloud/workspaces/variables.html#terraform-variables) values by providing any number of [files ending in `.auto.tfvars`](/docs/language/values/variables.html#variable-files) to workspaces that use Terraform 0.10.0 or later. When you trigger a run, Terraform automatically loads and uses the variables defined in these files. If any variable from the workspace has the same key as a variable in the file, the workspace variable overwrites variable from the file.

You can only do this with files ending in `auto.tfvars`; Terraform Cloud does not automatically recognize other types of `.tfvars` files. You can only apply other types of `.tfvars` files [on the command line](#run-specific-variables) for each run.
```